hivemind/client/averaging/matchmaking.py

```diff
@@ -10,6 +10,7 @@
 import asyncio
 
 import grpc
+from grpc._cython.cygrpc import InternalError
```

Comment:
```suggestion
```

nit: `InternalError` looks like a built-in error, so I suggest to remove this line and use the full name `grpc._cython.cygrpc.InternalError` instead.